packages/core-js/utils/deploymentHelper.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+
+function getTarget({ deploymentData, contract }) {
+  if (contract) return deploymentData.targets[contract];
+  else return deploymentData.targets;
+}
+
+function getSource({ deploymentData, contract }) {
+  if (contract) return deploymentData.sources[contract];
+  else return deploymentData.sources;
+}
+
+function getDeploymentFilePath({ artifactsPath, network, instance, jsonName }) {
+  console.log({ artifactsPath, network, instance, jsonName });
+  return path.join(artifactsPath, network, instance, jsonName);
+}
```

Comment:
nit: linter not picking this up?

Reply:
I think linter doesn't find any warning here :) Good catch

Reply:
Fixed